.storybook/global.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700;800&display=swap");
```

Comment:
I needed to modify this because i noticed that only weights of 400 and 600 worked. I mean if i set 800 it was ignored and was boldness of 600 same for values less than 600 that were ultimately equivalent to 400

Reply:
Good catch!